python/lsst/daf/butler/cli/opt/options.py

```diff
@@ -103,3 +103,12 @@ def makeCollectionType(context, param, value):
 verbose_option = MWOptionDecorator("-v", "--verbose",
                                    help="Increase verbosity.",
                                    is_flag=True)
+
+
+file_override_options = MWOptionDecorator("--options-file", "-@",
```

Comment:
I think the name should be `options_file_option`; basically `_option` indicates that it's an option (as opposed to an argument) and `options_file_` matches the name of the flag.